docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -227,6 +227,10 @@ implementation is used:
   - Number of threads used for retrieving metadata. Currently, only table loading 
     is parallelized.
   - `8`
+* - `iceberg.bucket-execution`
+  - Enable bucket-aware execution. This allows the engine to use physical
+    bucketing information to reduce data exchanges in queries.
```

Comment:
```suggestion
    bucketing information to reduce data exchanges in queries and there improve performance.
```

Reply:
Also .. should we add something about the fact that it automatically disables itself if necessary for a specific query and you should therefore nearly never set this to false..

Reply:
That's true for almost any optimization that we enable by default, so it probably doesn't need to be uniquely clarified here